mmcv/runner/hooks/logger/pavi.py

```diff
@@ -1,11 +1,26 @@
 # Copyright (c) Open-MMLab. All rights reserved.
+import numbers
 import os.path as osp
 
+import numpy as np
+import torch
+
 from mmcv.runner import master_only
 from ..hook import HOOKS
 from .base import LoggerHook
 
 
+def is_scalar(val, np_valid=True, torch_valid=True):
```

Comment:
- Add docstring here.
- `np_valid` and `torch_valid` may be renamed to `include_np`, `include_torch`.